src/coresyftools/coresyftools/tool.py

```diff
@@ -38,9 +59,27 @@ def __init__(self, run_script_file_name):
             self.context_directory, self.MANIFEST_FILE_NAME)
```

Comment:
Is there a reason we define the `MANIFEST_FILE_NAME` outside context of `__init__`?

Reply:
Yes, this way it can (and I think it already is) be directly imported from other modules.